analyzer/src/funTest/assets/projects/synthetic/dotnet-expected-output.yml

```diff
@@ -16,205 +16,40 @@ project:
     path: "<REPLACE_PATH>"
   homepage_url: ""
   scopes:
-  - name: "net45"
```

Comment:
We can recreate a scope now that nuget-inspector headers has a "project_framework" attribute